package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "ontologies",
+  "version": "1.0.0",
+  "description": "A repository for tracking work done on ontologies at Nesta. Mainly for creating issues on GitHub and saving interesting queries, as well as documenting the work and progress being made on Ontologies and using Linked Data sources.",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Here we need to have an npm script that runs `src/bin/OSM/download_latest_lgd_releases.sh`